OpenRA.Mods.Common/Orders/DeployOrderTargeter.cs

```diff
@@ -17,7 +17,7 @@ namespace OpenRA.Mods.Common.Orders
 {
 	public class DeployOrderTargeter : IOrderTargeter
 	{
-		readonly Func<string> cursor;
+		readonly Func<string> cursor; // TODO: Add [CursorReference]
```

Comment:
Same here.

Reply:
This one is easily fixable. Remove the `public DeployOrderTargeter(string order, int priority)` ctor and add cursor definitions to `PrimaryBuilding` and `MadTank`.